keps/2076-kueuectl/kep.yaml

```diff
@@ -0,0 +1,28 @@
+title: Kueuectl
+kep-number: 2076
+authors:
+  - "@mwielgus"
+status: implementable
+creation-date: 2024-04-25
+reviewers:
+  - "@alculquicondor"
+approvers:
+  - "@alculquicondor"
+replaces:
+  - "/keps/487-kubectl-plugin"
```

Comment:
Please update KEP's yaml to say `replaced` and leave a note there referencing this KEP.

Reply:
Done.